src/main/java/pl/agh/edu/master_diet/core/model/database/Browsable.java

```diff
@@ -0,0 +1,27 @@
+package pl.agh.edu.master_diet.core.model.database;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+
+@Getter
+@Setter
+@MappedSuperclass
+@NoArgsConstructor
+@AllArgsConstructor
+public abstract class Browsable {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false)
+    private String name;
+
+    @Column(nullable = false, columnDefinition = "int default 0")
+    private Integer approvals;
```

Comment:
You sure that this class should contain approvals? It's clear for the Product, but Activity also extends this class, does Activity require approvals?

Reply:
For me, activities should have approvals to the same as recipes later. We are going to let users add their own activities.

Reply:
ok